src/SelfScreener/HowAreYouFeeling.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import { useNavigation } from "@react-navigation/native"
+import {
+  Image,
+  ImageSourcePropType,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from "react-native"
+
+import { SelfScreenerStackScreens } from "../navigation"
+import { Typography, Spacing, Iconography, Colors, Outlines } from "../styles"
+import { GlobalText } from "../components"
+import { Images } from "../assets"
+
+const HowAreYouFeeling: FunctionComponent = () => {
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+
+  const handlePressGood = () => {
+    navigation.navigate(SelfScreenerStackScreens.AsymptomaticFlowIntro)
+  }
+  const handlePressNotWell = () => {
+    navigation.navigate(SelfScreenerStackScreens.EmergencySymptomsQuestions)
+  }
+
+  return (
+    <View style={style.contentContainer}>
```

Comment:
This should probably be a `ScrollView` to account for the possibility of VERY LARGE TEXT SIZES.